client/sidebar/sections/Omnichannel.tsx

```diff
@@ -34,8 +34,8 @@ const OmnichannelSection = (props: typeof Box): ReactElement => {
 
 	const voipCallIcon = {
 		title: !registered ? t('Enable') : t('Disable'),
-		color: !registered ? 'success' : undefined,
-		icon: !registered ? 'phone' : 'phone-disabled',
+		color: registered ? 'success' : undefined,
+		icon: registered ? 'phone' : 'phone-disabled',
```

Comment:
@MartinSchoeler Question about the icon. With the logic above we shall see
1. When registered, green phone. 
2. When clicked on green phone, unregistered and the icon changed with greyed striked out phone.

I am really confused about the UX here. if I change the condition as follows
icon: **!**registered ? 'phone' : 'phone-disabled',

1. This will result in striked out phone with green when registered,
2. When clicked, it will unregister and phone icon with grey.
What is the usual practice for the icons?

Reply:
```
color: registered ? 'success' : undefined,
icon: registered ? 'phone' : 'phone-disabled',
```

I think this is fine now

this button has 2 functions, toggling the status of the agent and displaying his current status (enabled/disabled)

As it is now, it should work out